### ISSUE TITLE
ask for resnet34.caffemodel

### ISSUE THREAD

**Issue by 408550969** [xieguotian/ResNet#1]
Sorry to bother you,I trained the resnet34 on Imagenet with the shortest side is 256, but the accuracy is only 67%. I can't reach 73% of the paper. Can you share your resnet34.caffemodel? Thanks!

**Comment by xieguotian** [comment]
My implementation of ResNet34 is 72.9%,  a little lower. And the model is based on my own implementation of caffe, not supportting the offical caffe, because I use a batchnormalization layer of my own.So it's not easy to use my model, and maybe you should build my own caffe to run it.  Do you still need it? If so, please give me your email, and I send my model to you.

**Comment by 408550969** [comment]
Well, thank you.
